samples/wifi/throughput/README.rst

```diff
@@ -0,0 +1,308 @@
+.. _wifi_throughput_sample:
+
+Wi-Fi: Throughput
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Throughput sample demonstrates how to measure the throughputs for Nordic Semiconductor's Wi-Fi® chipsets.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+The sample demonstrates how to measure network performance using **zperf** tool in different profiles.
+Using this sample, the development kit can associate with an access point.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/wifi/throughput`
+
+.. include:: /includes/build_and_run_ns.txt
+
+Currently, the nRF7002 DK + QSPI configuration is supported.
+
+To build for the nRF7002 DK, use the ``nrf7002dk_nrf5340_cpuapp`` build target.
+The following is an example of the CLI command:
+
+.. code-block:: console
+
+   west build -b nrf7002dk_nrf5340_cpuapp
+
+To build for the nRF7002 DK with different profiles for Station mode, use the following CLI commands:
```

Comment:
Can you add a phrase and a link to where the different profiles are defined? That is, on the corresponding NCS guide?